front-end/src/screens/OnChain/query.ts

```diff
@@ -28,14 +28,21 @@ query LatestProposalPosts {
     onchain_link {
       id
       onchain_proposal_id
-      onchain_referendum_id
       onchain_proposal(where: {}) {
         id
         proposalStatus {
           id
           status
         }
       }
+      onchain_referendum_id
```

Comment:
This is a "proposal" query, it shouldn't contain anything related to referenda.Actually, this query is not used anymore, this is the query related to the unused useLatestProposalPosts mentionned in HomeContent. This file should be deleted.